src/dll/edbgEcmdDll.C

```diff
@@ -1617,9 +1619,9 @@ uint32_t dllPutMemProc(const ecmdChipTarget & i_target, uint64_t i_address, uint
       blockSize = 8;
     }
 
-    rc = adu_putmem_io(adu_target, i_address, buf, i_bytes, blockSize);
```

Comment:
The same comment about a single instance of the function call from getmem applies here as well.